test/ejabberd_tests/tests/amp_SUITE.erl

```diff
@@ -238,20 +238,22 @@ notify_deliver_to_online_user_test(Config) ->
 notify_deliver_to_online_user_bare_jid_test(Config) ->
     escalus:fresh_story(
       Config, [{alice, 1}, {bob, 1}],
-      fun(Alice,Bob) ->
+      fun(Alice, Bob) ->
               %% given
               Rule = {deliver, direct, notify},
               Rules = rules(Config, [Rule]),
               BobsBareJid = escalus_client:short_jid(Bob),
-              Msg = amp_message_to(BobsBareJid, Rules, <<"One of your resources needs to get this!">>),
+              Msg = amp_message_to(BobsBareJid, Rules,
+                                <<"One of your resources needs to get this!">>),
               %% when
               client_sends_message(Alice, Msg),
               % then
               case lists:member(Rule, Rules) of
                   true -> client_receives_notification(Alice, BobsBareJid, Rule);
                   false -> ok
               end,
-              client_receives_message(Bob, <<"One of your resources needs to get this!">>),
+              client_receives_message(Bob,
+                                <<"One of your resources needs to get this!">>),
```

Comment:
Actually I see this text is the same in both places. Let's assign it to a var and use the var.